client/tailwind.config.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
         blueGray: '#E0E5ED',
         lightBlue: '#325184',
         lightBrown: '#cb8075',
+        tableGray: '#D7D6D6',
```

Comment:
```suggestion
```

I seems the color's already been declared down below, so please remove this one use the existing one